file-formats/edck/type/zif_aff_edck_v1.intf.abap

```diff
@@ -0,0 +1,74 @@
+INTERFACE zif_aff_edck_v1
+  PUBLIC.
+
+  TYPES ty_check_classification TYPE c LENGTH 1.
+  TYPES ty_check_category TYPE c LENGTH 6.
+
+  TYPES:
+    BEGIN OF ty_attributes,
+      "! <p class="shorttext">Check Category</p>
+      "! Consistency check category
+      "! $values {@link zif_aff_edck_v1.data:co_check_category}
+      "! $default {@link if_aff_edck_v1.data:co_check_category.exist}
+      "! $required
```

Comment:
```suggestion
```
If you have set a default value, you don't need the `$required` annotation, as the value can't be empty (because it is automatically set to the default).